src/test/java/uk/gov/companieshouse/chdorderconsumer/controller/HealthcheckControllerIntegrationTest.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.companieshouse.chdorderconsumer.controller;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
```

Comment:
We should not be using * imports , These can be picked up by static code analysis tools (Sonar) .

Reply:
corrected